integration/ui-schema.spec.ts

```diff
@@ -121,7 +121,8 @@ test("form should render correctly according to uiSchema", async (t) => {
   // one at the root level, the other nested in `misc`
   const docContents2 = JSON.parse(readFileSync(filePath2, "utf8"));
   // Need stringify first, then parse. Without it, getData() alone's return type is never.
-  const docContentsUnwrapped2 = JSON.parse(JSON.stringify(getData({ data: docContents2.data })));
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const docContentsUnwrapped2 = getData(docContents2) as any;
   await t.expect(docContentsUnwrapped2.misc.uiSchema.remarks).contains({
```

Comment:
see if can type?

Reply:
i rmb y this need any type, it is already using OpenAttestationDocument type but because its a special test case for misc and uiSchema in this test thats y need this as any.